pg_service_parser/ui/pg_service_dialog.ui

```diff
@@ -260,6 +244,59 @@
          </property>
         </spacer>
        </item>
+       <item row="1" column="1">
+        <widget class="QPushButton" name="btnCopyService">
+         <property name="text">
+          <string>Copy service</string>
+         </property>
+        </widget>
+       </item>
+       <item row="3" column="0" colspan="2">
+        <widget class="QGroupBox" name="groupBox">
+         <property name="title">
+          <string>Shortcuts</string>
```

Comment:
Perhaps we could add more details on what _Shortcuts_ are for or how they are to be used?

For instance:
	"Shortcuts (to be listed in plugin's toolbar and menu)"